CMakeLists.txt

```diff
@@ -91,6 +91,22 @@ install(DIRECTORY include/openssl
         PATTERN boringssl_prefix_symbols_nasm.inc EXCLUDE
 )
 
+if (TEST_SYSGENID_PATH)
+  message(STATUS "Setting AWSLC_SNAPSAFE_TESTING=1")
+  add_definitions(-DAWSLC_SNAPSAFE_TESTING=1)
+  message(STATUS "Setting AWSLC_SYSGENID_PATH=${TEST_SYSGENID_PATH}")
+  add_definitions(-DAWSLC_SYSGENID_PATH=\"${TEST_SYSGENID_PATH}\")
+
+  if (SYSGENID_PATH)
+    message(FATAL_ERROR "Cannot set both TEST_SYSGENID_PATH and SYGENID_PATH")
+  endif()
+endif()
+
+if (SYSGENID_PATH)
```

Comment:
Is this necessary? You can define `AWSLC_SYSGENID_PATH` in your build if you want to.

Reply:
Ok, I'll remove this build option.